pkg/sqlite/migrator.go

```diff
@@ -111,6 +111,28 @@ func (m *SQLMigrator) InitMigrationVersion() error {
 	return nil
 }
 
+// MigrateUp is a wrapper around golang-migrate's Up. Up
+// looks at the currently active migration version and will
+// migrate all the way up (applying all up migrations).
+func (m *SQLMigrator) MigrateUp() error {
+	instance, err := sqlite3.WithInstance(m.db, &sqlite3.Config{})
```

Comment:
This will make the migrator sqlite-only. 

If we change 
```go
type SQLMigrator struct {
	db             *sql.DB
	migrationsPath string
	generated bool
}
```

to 

```go
type SQLMigrator struct {
	db             *database.Driver
	migrationsPath string
	generated bool
}
```
then we can use the migrator for any db?

Reply:
We can make this change, but the golang-migrate needs that instance object to understand which database and explicitly support it. IMO we should probably revisit this later if we want to abstract the migrator to understand multiple database types.

Reply:
+1 with Kevin.

Reply:
> golang-migrate needs that instance object to understand which database and explicitly support it.

I don't think this is the case, but we can revisit it in the future.

I do think we should be careful to pass in a `DatabaseName` to `sqlite3.Config`, right?

Reply:
> I do think we should be careful to pass in a DatabaseName to sqlite3.Config, right?

@ecordell I've updated the PR to pass in a `DatabaseName`. Why do you say `careful` though? I didn't seem to find any "best practice guide" around passing the database name in the docs so was curious.